composer.json

```diff
@@ -67,8 +68,7 @@
         "google/cloud-tools": "^0.12.0",
         "opis/closure": "^3.0",
         "swaggest/json-schema": "^0.12.0",
-        "wikimedia/avro": "^1.9",
-        "monolog/monolog": "<2.3.0"
```

Comment:
Was dropping this intentional?

Reply:
Yes... But it's unrelated to this PR. It was added a month or so back to fix a bug we were having in monolog that has since been fixed, so I just removed it here.